api/test/e2e/route_with_vaild_remote_addr.go

```diff
@@ -24,7 +24,7 @@ import (
 func TestRoute_with_vaild_remote_addr(t *testing.T) {
 	tests := []HttpTestCase{
 		{
-			caseDesc: "add route with vaild remote_addr",
+			caseDesc: "add route with vaild remote_address",
```

Comment:
`vaild` -> `valid`
typo, please fix other similar points and the file name.

Reply:
and I think we use `remote_addr` is better

Reply:
fixed.